hack/force-cert-retation.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -xe
+
+KUBECTL=${KUBECTL:-kubectl}
+POD_NAMESPACE=${POD_NAMESPACE:-kubemacpool-system}
```

Comment:
Let's prevent people from using these defaults by setting them to empty values. People may get used to the fact that this script works well with totally unrelated KMP and be surprised when it stops working. It's better to leave it empty, add a check that verifies that the variable was set and README with an example on how to set them.

Reply:
I'll add help section to the script and reference it on README.md